uber/templates/groups/history.html

```diff
@@ -30,14 +30,36 @@
 {% endfor %}
 </table>
 
+    {% if group.stripe_transactions %}
```

Comment:
Whoops, this is from https://github.com/magfest/ubersystem/pull/2471. There will probably be a merge conflict when we merge that PR, I'll handle it when it comes up.